content/webapp/services/prismic/transformers/series.ts

```diff
@@ -1,20 +1,75 @@
 import * as prismic from '@prismicio/client';
 
+import { ImageType } from '@weco/common/model/image';
 import {
   SeasonsDocument as RawSeasonsDocument,
   SeriesDocument as RawSeriesDocument,
   StandfirstSlice as RawStandfirstSlice,
+  WebcomicSeriesDocument as RawWebcomicSeriesDocument,
 } from '@weco/common/prismicio-types';
 import { transformTimestamp } from '@weco/common/services/prismic/transformers';
+import { transformImage } from '@weco/common/services/prismic/transformers/images';
 import { isNotUndefined } from '@weco/common/utils/type-guards';
 import { ArticleScheduleItem } from '@weco/content/types/article-schedule-items';
 import { Series, SeriesBasic } from '@weco/content/types/series';
 import { getSeriesColor } from '@weco/content/utils/colors';
 
-import { asTitle, transformGenericFields, transformSingleLevelGroup } from '.';
+import {
+  asText,
+  asTitle,
+  isGenericDocWithMetaDescription,
+  isGenericDocWithPromo,
+  transformGenericFields,
+  transformSingleLevelGroup,
+} from '.';
 import { transformContributors } from './contributors';
+import { transformImagePromo } from './images';
 import { transformSeason } from './seasons';
 
+export function transformWebcomicSeries(
+  doc: RawWebcomicSeriesDocument
+): Series {
+  const { data } = doc;
+  const promo = isGenericDocWithPromo(doc)
+    ? transformImagePromo(data.promo)
+    : undefined;
+
+  const primaryPromo =
+    isGenericDocWithPromo(doc) && data.promo.length > 0
+      ? data.promo
+          .filter((slice: prismic.Slice) => slice.primary.image)
+          .find(_ => _)
+      : undefined;
+
+  const image: ImageType | undefined = primaryPromo
+    ? transformImage(primaryPromo.primary.image)
+    : undefined;
+
+  const metadataDescription = isGenericDocWithMetaDescription(doc)
+    ? asText(data.metadataDescription)
+    : undefined;
+
+  const contributors = transformContributors(doc);
```

Comment:
Would using `const genericFields = transformGenericFields(document)` and passing that as we do for series not do for a lot of these? or maybe it was overkill. We'll be rid of webcomic series soon I hope :')

Reply:
the webcomic-series type in Prismic doesn't have all the bits transformGenericFields is expecting so I did this rather than try and account for the missing bits in the function - like you said hopefully they'll be gone soon